notebooks.go

```diff
@@ -232,23 +232,41 @@ func (s *server) GetNotebooks(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *server) handleVolume(ctx context.Context, req volumerequest, notebook *kubeflowv1.Notebook) error {
+	var pvc = corev1.PersistentVolumeClaim{}
 	if req.Type == VolumeTypeNew {
-		// Create the PVC
-		pvc := corev1.PersistentVolumeClaim{
-			ObjectMeta: v1.ObjectMeta{
-				Name:      req.Name,
-				Namespace: notebook.Namespace,
-			},
-			Spec: corev1.PersistentVolumeClaimSpec{
-				AccessModes: []corev1.PersistentVolumeAccessMode{req.Mode},
-				Resources: corev1.ResourceRequirements{
-					Requests: corev1.ResourceList{
-						corev1.ResourceStorage: req.Size,
+		if _, ok := notebook.Labels["notebook.statcan.gc.ca/protected-b"]; ok {
+			pvc = corev1.PersistentVolumeClaim{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      req.Name,
+					Namespace: notebook.Namespace,
+					Labels:    map[string]string{"data.statcan.gc.ca/classification": "protected-b"},
+				},
```

Comment:
Is there a way in go to just have this part conditional?

Reply:
I don't think so?